test/fixtures/test-runner/output/describe_it.js

```diff
@@ -334,44 +334,51 @@ describe('timeouts', () => {
 
 describe('successful thenable', () => {
   it('successful thenable', () => {
-    let thenCalled = false;
     return {
       get then() {
-        if (thenCalled) throw new Error();
-        thenCalled = true;
         return (successHandler) => successHandler();
       },
     };
   });
 
   it('rejected thenable', () => {
-    let thenCalled = false;
     return {
       get then() {
-        if (thenCalled) throw new Error();
-        thenCalled = true;
         return (_, errorHandler) => errorHandler(new Error('custom error'));
       },
     };
   });
 
-  let thenCalled = false;
   return {
     get then() {
-      if (thenCalled) throw new Error();
-      thenCalled = true;
       return (successHandler) => successHandler();
     },
   };
 });
 
 describe('rejected thenable', () => {
-  let thenCalled = false;
   return {
     get then() {
-      if (thenCalled) throw new Error();
-      thenCalled = true;
       return (_, errorHandler) => errorHandler(new Error('custom error'));
     },
   };
 });
```

Comment:
does this mean this change is breaking change? as in order to check if describe returns a promise we must access the `then` property, we may avoid that if we would wrap the function result in other promise and use that `then`